bot.py

```diff
@@ -829,7 +829,25 @@ def command_perm(self, command_name: str) -> PermissionLevel:
 
             thread = await self.threads.find(channel=ctx.channel)
             if thread is not None:
-                if self.config.get("reply_without_command"):
+                try:
+                    reply_without_command = strtobool(
+                        self.config["reply_without_command"]
+                    )
+                except ValueError:
+                    reply_without_command = self.config.remove("reply_without_command")
+
+                try:
+                    anon_reply_without_command = strtobool(
+                        self.config["anon_reply_without_command"]
+                    )
+                except ValueError:
+                    anon_reply_without_command = self.config.remove(
+                        "anon_reply_without_command"
+                    )
+
+                if anon_reply_without_command:
+                    await thread.reply(message, anonymous=True)
+                elif reply_without_command:
```

Comment:
The syntax changed for config parsing, as you can see to the [left](https://github.com/kyb3r/modmail/pull/388/files#diff-f289368d1ae6d2d58bec9f799832d932L832), just do:
    if self.config.get("reply_without_command"):
        await thread.reply(message)
    elif self.config.get("anon_reply_without_command"):
        await thread.reply(message, anonymous=True)
    else:
        ...